package.json

```diff
@@ -36,7 +36,7 @@
     "husky": "8.0.3",
     "jest": "29.6.1",
     "lint-staged": "13.2.3",
-    "prettier": "3.0.0",
+    "prettier": "2.8.8",
```

Comment:
Had to downgrade prettier because of: https://github.com/prettier/eslint-plugin-prettier/issues/562